docs/guide/snaps.md

```diff
@@ -82,25 +82,35 @@ Let's extend the functionality of MetaMask and build the wallet experience of th
 ### Prerequisites
 
 - Up-to-date Chromium or Firefox browser
-- [Node.js 16.0](https://nodejs.org/) (we recommend using [nvm](https://github.com/nvm-sh/nvm))
+- [Node.js](https://nodejs.org/)
```

Comment:
Is there a reason not to specify a minimum Node version?

Reply:
FYI Snaps will not work with <16

Reply:
From reading the docs the first time, it felt like a lot of information to take in in order to get started. 

 I wanted to strike a balance with making information available but not overwhelming people with a big cognitive load. I see two cases that can happen here if a user is not on 16+:
 
 - Experienced users will see the `.nvmrc` file and adjust themselves quickly
 - Less experienced users will revert back here and stumble on the `<blockquote>` explaining the exact information I am removing here.